apps/recovery-relay/lib/wallets/types.ts

```diff
@@ -28,6 +28,7 @@ export type AccountData = {
   extraParams?: Map<string, any>;
   endpoint?: string;
   insufficientBalance?: boolean;
+  insufficientFeeBalance?: boolean;
```

Comment:
Unclear, please update to the below and update references

```suggestion
  insufficientBalanceForTokenTransfer?: boolean;
```